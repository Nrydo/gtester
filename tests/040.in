3379443
45307137
116432231