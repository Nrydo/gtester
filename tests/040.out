3630450574354419